arkouda/pdarrayclass.py

```diff
@@ -1168,6 +1188,38 @@ def info(pda : Union[pdarray, str]) -> str:
         raise TypeError("info: must be pdarray or string".format(pda))
         return generic_msg(cmd="info", args="{}".format(pda))
 
+def list_registry() -> list[str]:
+    """
+    Return a list containing the names of all registered objects
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    list
+        List of all object names in the registry
+
+    Raises
+    ------
+    RuntimeError
+        Raised if there's a server-side error thrown
+    """
+    registered_list: list[str] = []
+
+    if info('__RegisteredSymbols__') == '__EMPTY_REGISTRY__':
+        return registered_list
+
+    registered_object: str
+    for registered_object in filter(None, info('__RegisteredSymbols__').split('\n')):
```

Comment:
I noticed `__RegisteredSymbols__` is used twice here; does it make sense to import it directly from `arkouda.client` and use the defined form vs the string literal?

Reply:
I would like you to use ak.RegisteredSymbols instead of the string literal.

Reply:
@glitch and @mhmerrill, I agree completely. I'll change this in my next commit